src/Augmentation/AugmentationHelpers.jsx

```diff
@@ -361,8 +361,8 @@ function initAugmentations() {
 
     const GrapheneBionicSpine = new Augmentation({
         name:AugmentationNames.GrapheneBionicSpine, repCost:1.625e6, moneyCost:6e9,
-        info:"An upgrade to the Bionic Spine augmentation. It fuses the implant with an advanced graphene " +
-             "material to make it much stronger and lighter.<br><br>" +
+        info:"An upgrade to the 'Bionic Spine' augmentation. The spine is fused with graphene " +
```

Comment:
I think we need either a comma or some "connecting-word" after "graphene. Something like:

"The spine is fused with graphene, enhancing durability and supercharging all body functions."

or

"The spine is fused with graphene which enhances durability and supercharges all body functions"

or something along those lines